src/nidcpower/metadata/attributes_addon.py

```diff
@@ -3,8 +3,26 @@
 
 # We are not code genning attributes that have been marked as obsolete prior to the initial
 # Python API bindings release
-attributes_codegen_method = {
+
+# Disconnect attribute from enum, either because associated enum is just values, or because it 
```

Comment:
I think a newcomer will have trouble figuring out what "associated enum is just values" means.
Suggestion: "Enums in extracted metadata are actually just constants."

Reply:
Used same wording as in #717